test/run_test.py

```diff
@@ -1130,7 +1130,7 @@ def get_selected_tests(options):
         selected_tests = [tname for tname in selected_tests if tname in FUNCTORCH_TESTS]
 
     if options.mps:
-        selected_tests = ["test_mps", "test_metal"]
+        selected_tests = ['test_mps', 'test_metal', 'test_modules']
```

Comment:
Please use double quotes, per https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings
```suggestion
        selected_tests = ["test_mps", "test_metal", "test_modules"]
```